pkg/compiler/consts.go

```diff
@@ -13,11 +14,12 @@ import (
 )
 
 type ConstInfo struct {
-	File     string
-	Consts   []string
-	Includes []string
-	Incdirs  []string
-	Defines  map[string]string
+	Pos       ast.Pos
```

Comment:
Position shouold be per-const.

Reply:
FWIW we make them up in a few places

https://github.com/google/syzkaller/blob/d265efd8b4153abba6f7b44082305976371beaad/sys/syz-extract/openbsd.go#L74-L80

https://github.com/google/syzkaller/blob/d265efd8b4153abba6f7b44082305976371beaad/sys/syz-extract/netbsd.go#L75-L90

Reply:
I've refactored `info.Consts`.